### ISSUE TITLE
how to get the gradient which loss function with respect to ReLU layer output feature?

### ISSUE THREAD

**Issue by wang-qun-sen** [apache/incubator-mxnet#11941]
when i read the paper <<LDMNet: Low Dimensional Manifold Regularized Neural Networks>>,
https://arxiv.org/abs/1711.06246?context=cs
 i got a different regularization iterm which come from the output feature of middle layer Relu. i know it is not like the weight decay , and i have to modify the backward of the gradient, but how to do this ? 
![image](https://user-images.githubusercontent.com/25158231/43443532-b6846756-94d3-11e8-966a-334fba061442.png)

**Comment by Roshrini** [comment]
@wang-qun-sen Thanks for trying it out with mxnet.
For how-to and usability questions, you can use MXNet discuss forum - https://discuss.mxnet.io/ to involve wider community.
We want to use Github issues mainly for bugs and feature request. Feel free to reopen if closed in error.

@sandeep-krishnamurthy Can you please label- Question and close this issue?
